crates/runtime/src/accelerated_table/refresh.rs

```diff
@@ -466,7 +488,7 @@ mod tests {
             MemTable::try_new(schema, vec![vec![batch]]).expect("mem table should be created"),
         ) as Arc<dyn TableProvider>;
 
-        let refresh = Refresh::new(None, None, None, None, RefreshMode::Full, None, None);
+        let refresh = Refresh::new(None, None, None, None, None, RefreshMode::Full, None, None);
```

Comment:
Nit: we can add `with_jitter` fn so we don't need to pass None everywhere / tests